web/app/js/components/TopRoutesTable.jsx

```diff
@@ -0,0 +1,75 @@
+import { metricToFormatter, numericSort } from './util/Utils.js';
+import BaseTable from './BaseTable.jsx';
+import PropTypes from 'prop-types';
+import React from 'react';
+import SuccessRateMiniChart from './util/SuccessRateMiniChart.jsx';
+import _ from 'lodash';
+
+const routesColumns = [
+  {
+    title: "Route",
+    dataIndex: "route",
+    render: d => {
+      return _.isEmpty(d.route) ? "UNKNOWN" : d.route;
+    },
+    sorter: (a, b) => (a.route || "").localeCompare(b.route)
```

Comment:
This sorts `UNKNOWN` first by default. Any way to sort it to the bottom?

![screen shot 2018-11-27 at 3 43 02 pm](https://user-images.githubusercontent.com/236915/49118925-274b6b80-f25b-11e8-949a-26e99d663a97.png)

Reply:
fixed!